src/sonic-config-engine/minigraph.py

```diff
@@ -627,7 +627,13 @@ def parse_xml(filename, platform=None, port_config_file=None):
         'hostname': hostname,
         'hwsku': hwsku,
         'type': current_device['type']
-        }}
+        },
+        'x509': {
+        'server_crt': '/etc/sonic/telemetry/streamingtelemetryserver.cer',
+        'server_key': '/etc/sonic/telemetry/streamingtelemetryclient.key',
+        'ca_crt': '/etc/sonic/telemetry/dsmsroot.cer'
```

Comment:
What if we need other certificates on the SONiC box? Maybe the keys are too generic: "server_crt", "server_key", "ca_crt". Maybe we need to prefix them with the component they're used for, e.g., "telemetry_server_crt", "telemetry_server_key", "telemetry_ca_crt", etc.

@lguohan: Do you agree?

Reply:
Hi Joe, such keys are used in telemetry contrainers and already defined: 
 https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/telemetry.sh

If we need change, we need change this place as well. It will break other partner using this feature now, if we device to use other key names.

Reply:
Ah. Good to know.

@lguohan: Should we merge this as-is and deal with making changes to support other certificates if and when we need them?

Reply:
@hui-ma: I think we can go ahead and merge this as-is for now, but I think we need to go back and think about a way to restructure this in order for SONiC to manage multiple keys/certs.

Reply:
sounds good to me